ClassicImplementation/App.tsx

```diff
@@ -1,30 +1,41 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * Generated with the TypeScript template
- * https://github.com/react-native-community/react-native-template-typescript
- *
- * @format
- */
-
-import React from 'react';
-import { Navigation } from './src/app/navigation';
-import { useAppInit } from './src/app/useAppInit';
-import { PrivacyManagerProvider } from './src/shared/views/PrivacyManager/usePrivacyManagerModal';
+import React, { useEffect } from 'react';
+import { useAppInit } from './src/useAppInit';
+import Contentsquare from '@contentsquare/react-native-bridge';
+import { StyleSheet, Text, View } from 'react-native';
+import { colors, gridUnit } from './src/constants';
 
 const App = () => {
   const { isLoadingComplete } = useAppInit();
 
+  useEffect(() => {
+    Contentsquare.optIn();
+    Contentsquare.send("Classic implementation screen view");
```

Comment:
J'ai peut-être une question de compréhension par contre, je remarque que sur l'ancienne implémentation, le choix de l'OptIn est 'sauvegardé' sur l'AsyncStorage et donc n'est pas trigger à chaque démarrage. Ici il va démarrer si l'utilisateur kill l'App et la relance right ? 
A mes yeux ça ne poserait pas de pb sauf que ça veut dire que ça génère un nouveau UserId() à chaque démarrage, est-ce que ça peut poser pb, bonne pratique à donner au user ou non ?  https://docs.contentsquare.com/react-native/#opt-in

Reply:
C'est une bonne remarque !
Du coup j'ai testé en appelant `ContentSquare.getUserId()` juste après `Contentsquare.optIn()` et le user id que je reçois est le même, même après avoir killé l'app.
Je pense que le SDK CS sauvegarde lui même le user id et si oui ou non l'utilisateur a opt in. Si tu appelles deux fois `optIn` d'affilée, je pense que le SDK ignore le second appel, ce qui est plus robuste.

Reply:
Trop cool ! au top alors :) !